io.catenax.fleet.diagnostic_data/2.0.0/DiagnosticData.ttl

```diff
@@ -0,0 +1,541 @@
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.1.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.1.0#> .
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.1.0#> .
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.1.0#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <urn:samm:io.catenax.fleet.diagnostic_data:2.0.0#> .
+@prefix ext-uuid: <urn:samm:io.catenax.shared.uuid:1.0.0#> .
+
+:DiagnosticData a samm:Aspect ;
+   samm:description "A Catena-X semantic data model for a list of vehicle diagnostic sessions."@en ;
+   samm:properties ( :diagnosticSessions [ samm:property :metaInformation; samm:optional true ] ) ;
+   samm:operations ( ) ;
+   samm:events ( ) .
+
+:diagnosticSessions a samm:Property ;
```

Comment:
Property and the referenced Characteristic should not have the same name

Reply:
fixed with next upload